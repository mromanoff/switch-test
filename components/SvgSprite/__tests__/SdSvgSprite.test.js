import { shallowMount } from '@vue/test-utils';
import SdSvgSprite from '../SvgSprite.vue';

describe('SdSvgSprite.vue', () => {
  const wrapper = shallowMount(SdSvgSprite);

  it('renders svg sprite', () => {
    expect(wrapper.is('svg')).toBe(true);
    expect(wrapper).toMatchSnapshot();
  });
});
