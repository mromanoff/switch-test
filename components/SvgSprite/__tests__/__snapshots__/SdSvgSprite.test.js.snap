// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SdSvgSprite.vue renders svg sprite 1`] = `
VueWrapper {
  "_emitted": Object {},
  "_emittedByOrder": Array [],
  "isFunctionalComponent": undefined,
}
`;
